docs/ja/reference/cli.md

```
@@ -5,70 +5,118 @@ outline: deep
+  
+  * [設定ファイル](#設定ファイル)
+    
+  * [プロファイル管理](#プロファイル管理)
+    
+  * [環境変数](#環境変数)
+    
+  * [オプション解決順序](#オプション解決順序)
+    
+* [認証](#認証)
+  
+  * [メール / パスワードログイン](#email--password-login)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)